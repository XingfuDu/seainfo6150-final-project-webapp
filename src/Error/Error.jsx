import React from 'react'
import classes from './Error.module.css';

const Error = () => {
	return (
		<div className={classes.wrapper}>
			<article className={classes.article}>
                <h1 className={classes.p}>404 Not Found</h1>
			</article>
		</div>
	);
};

export default Error;
