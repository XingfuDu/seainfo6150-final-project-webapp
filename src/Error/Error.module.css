.wrapper {
	width: 100%;
	min-height: 100%;
	background-color: #f2e9e4;
	padding: 15px;
	font-family: 'Lora', serif;
    box-sizing: border-box;
    display: flex;
    align-items: center;
    justify-content: center;
}
.article {
	min-height: 100%;
	font-size: 150%;
}


.p {
	color: #3d5a80;
}

