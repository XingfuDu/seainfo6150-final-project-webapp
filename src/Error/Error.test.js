import React from 'react';
import Error from './Error'; 
import { render } from "@testing-library/react";

describe('Error tests', () => {
	it('renders correctly', () => {
		const { container } = render(<Error/>);
		expect(container).toMatchSnapshot();
	});
});
