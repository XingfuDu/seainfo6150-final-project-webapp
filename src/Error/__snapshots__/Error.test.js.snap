// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error tests renders correctly 1`] = `
<div>
  <div
    class="wrapper"
  >
    <article
      class="article"
    >
      <h1
        class="p"
      >
        404 Not Found
      </h1>
    </article>
  </div>
</div>
`;
